tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_bug_1116566.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import SkipTest
+from marionette.by import By
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.email.app import Email
+from gaiatest.apps.system.regions.cards_view import CardsView
+
+
+class TestBug1116566(GaiaTestCase):
+    _menu_button_locator = (By.CSS_SELECTOR, '.msg-folder-list-btn')
+
+    def setUp(self):
+        try:
+            self.testvars['email']['IMAP']
+        except KeyError:
+            raise SkipTest('account details not present in test variables')
+
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+        # Open email app
+        self.email = Email(self.marionette)
+        self.email.launch()
+
+    def test_bug_1116566(self):
+        """ https://bugzilla.mozilla.org/show_bug.cgi?id=1116566 """
```

Comment:
Yeah, I agree. I don't even think that a special bug had to be created anyway, to make a gaia-ui test out of this. The gaia-ui test writing could just be done in the bug itself, as is normally done with automated test writing, in general.